$NetBSD: patch-ab,v 1.3 2014/02/01 04:12:04 markd Exp $

Make LUA optional (PR/43054). Make sure man pages are installed
at the right place.
Dont look for CppUnit as we dont want to build tests.

--- CMakeLists.txt.orig	2011-04-26 18:02:47.000000000 +0000
+++ CMakeLists.txt
@@ -340,7 +340,7 @@ ELSE(PNG_FOUND)
 ENDIF(PNG_FOUND)
 
 
-FIND_PACKAGE(CppUnit)
+# FIND_PACKAGE(CppUnit)
 
 IF(CppUnit_FOUND)
   MESSAGE("Found cppunit. Unit tests will be built.")
@@ -407,7 +407,9 @@ ELSE(DEFINED JPEG_RUNTIME_COMPATIBLE)
 	ENDIF(NOT WIN32)
 ENDIF(DEFINED JPEG_RUNTIME_COMPATIBLE)
 
-FIND_PACKAGE(LUA)
+IF(DEFINED LUA_ENABLED)
+	FIND_PACKAGE(LUA)
+ENDIF(DEFINED LUA_ENABLED)
 IF(LUA_FOUND)
 	# If we have lua, we can build podofoimpose.
 	MESSAGE("Lua found - PoDoFoImpose and PoDoFoColor will be built with Lua support")
@@ -485,7 +487,7 @@ SET(PODOFO_LIB
 #
 # Setup directories we will need
 #
-SET(MANDIR "share/man/")
+SET(MANDIR "man/")
 
 # Create the config file. It'll be appended to as the subdirs run though
 # then dependency information will be written to it at the end of the
